src/api/config/sumeragi-params.md

```diff
@@ -0,0 +1,195 @@
+# Sumeragi Parameters
+
+TODO Explain sumeragi module. Clarify that it consists of transactions and blocks gossiping, which are two separate
+processes. Explain how periods & gossip sizes might affect network load & speed of sync. Explain the pipeline of
+transactions and blocks, refer to consensus section and maybe some others.
+
+## `sumeragi.block_time`
+
+- **Type:** String or Number, [Duration](glossary#type-duration)
+- **Default:** 2 seconds
+
+Time since the round start[^1] by which a new block should be created regardless if there were enough transactions or
+not. Used to force block commits when there is a small influx of new transactions.
+
+A block might be created earlier if there is enough transactions in the [Queue](queue-params). The limit of transactions
+is configured by [`sumeragi.transactions_in_block`](#sumeragi-transactions-in-block).
+
+**Example:**
+
+```toml
+[sumeragi]
+block_time = "2s"
+```
+
+## `sumeragi.block_gossip_period`
+
+- **Type:** String or Number, [Duration](glossary#type-duration)
+- **Default:** 10 seconds
+
+The time interval between requests to peers for the most recent block.
```

Comment:
`requests to peers for the most recent block` does this mean requests *about* the most recent block?

Reply:
Yes. Request the most recent block from peers.

Reply:
then maybe

```suggestion
The time interval between sending requests to peers about the most recent block.
```